sorc/mdate.fd/mdate.f

```diff
@@ -1,38 +1,43 @@
-C> @file
-C>
-C> UPDATE A DATE GIVEN INCREMENT IN MINUTES
-C> @author KEYSER  @date 2001-09-20
-C>
-C> PROGRAM TO COMPUTE A VERIFYING DATE GIVEN AN INCREMENT IN
-C> MINUTES AND THE INITIAL DATE.
-C>
-C> PROGRAM HISTORY LOG:
-C> - 2001-09-20  KEYSER   BASED ON UTILITY PROGRAM NDATE EXCEPTS UPDATES
-C>                      BY MINUTES INSTEAD OF HOURS
-C>
-C>   INPUT ARGUMENT LIST
-C>   -  MINUTES  - INCREMENT IN MINUTES (MAY BE NEGATIVE)
-C>                MINUTES DEFAULTS TO ZERO.
-C>   -  IDATE    - INITIAL DATE IN YYYYMMDDHHMM FORMAT.
-C>                IDATE DEFAULTS TO THE CURRENT UTC DATE, HOUR AND
-C>                MINUTE.  FIRST CENTURY DATES WILL CAUSE ABORT AFTER
-C>                1 SEPT 1999.
-C>   OUTPUT ARGUMENT LIST:
-C>   -  MDATE    - VERIFYING DATE IN YYYYMMDDHHMM FORMAT.
-C>   EXIT STATES:
-C>   -  0      - SUCCESS
-C>   -  1      - FAILURE; INVALID ARGUMENT
-C>   -  2      - FAILURE; INCORRECT NUMBER OF ARGUMENTS
-C>
-C> SUBPROGRAMS CALLED:
-C>  - IARGC           GET NUMBER OF ARGUMENTS
-C>  - GETARG          GET ARGUMENT
-C>  - W3MOVDAT        RETURN A DATE FROM A TIME INTERVAL AND DATE
-C>  - W3UTCDAT        RETURN THE UTC DATE AND TIME
-C>  - ERRMSG          WRITE A MESSAGE TO STDERR
-C>  - ERREXIT         EXIT PROGRAM (W3LIB)
-C>
       PROGRAM MDATE
+C$$$  MAIN PROGRAM DOCUMENTATION BLOCK
+C
+C MAIN PROGRAM: MDATE        UPDATE A DATE GIVEN INCREMENT IN MINUTES
+C   PRGMMR: KEYSER           ORG: NP22        DATE: 2001-09-20
+C
+C ABSTRACT: PROGRAM TO COMPUTE A VERIFYING DATE GIVEN AN INCREMENT IN
+C   MINUTES AND THE INITIAL DATE.
+C
+C PROGRAM HISTORY LOG:
+C 2001-09-20  KEYSER   BASED ON UTILITY PROGRAM NDATE EXCEPTS UPDATES
+C                      BY MINUTES INSTEAD OF HOURS
+C
+C USAGE:      mdate [minutes [idate]]
+C   INPUT ARGUMENT LIST
+C     MINUTES  - INCREMENT IN MINUTES (MAY BE NEGATIVE)
+C                MINUTES DEFAULTS TO ZERO.
+C     IDATE    - INITIAL DATE IN YYYYMMDDHHMM FORMAT.
+C                IDATE DEFAULTS TO THE CURRENT UTC DATE, HOUR AND
+C                MINUTE.  FIRST CENTURY DATES WILL CAUSE ABORT AFTER
+C                1 SEPT 1999.
+C   OUTPUT ARGUMENT LIST:
+C     MDATE    - VERIFYING DATE IN YYYYMMDDHHMM FORMAT.
+C   EXIT STATES:
+C     0      - SUCCESS
+C     1      - FAILURE; INVALID ARGUMENT
+C     2      - FAILURE; INCORRECT NUMBER OF ARGUMENTS
+C
+C SUBPROGRAMS CALLED:
+C   IARGC           GET NUMBER OF ARGUMENTS
+C   GETARG          GET ARGUMENT
+C   W3MOVDAT        RETURN A DATE FROM A TIME INTERVAL AND DATE
+C   W3UTCDAT        RETURN THE UTC DATE AND TIME
+C   ERRMSG          WRITE A MESSAGE TO STDERR
+C   ERREXIT         EXIT PROGRAM (W3LIB)
+C
+C ATTRIBUTES:
+C   LANGUAGE: FORTRAN 90
+C
+C$$$
```

Comment:
The changes to this file remove the doxygen formatting. Unless new material was added to the comments, the old, doxygen-enhanced version should be kept.